test/Akkatecture.Tests/UnitTests/Commands/CommandTests.cs

```diff
@@ -10,7 +10,7 @@ namespace Akkatecture.Tests.UnitTests.Commands
     public class CommandTests
     {
         [Fact]
-        public void InstantiatingCommand_WithValidInput_ThrowsException()
+        public void InstantiatingCommand_WithValidInput_ThrowsException_Succeeds()
```

Comment:
Can you take out the "ThrowsExceptions" part of the name? There are none raised in this particular test.